Makefile

```diff
@@ -36,11 +36,11 @@ unit-tests: dependencies cmd/wego/ui/run/dist/index.html
 debug:
 	go build -ldflags $(LDFLAGS) -o bin/$(BINARY_NAME) -gcflags='all=-N -l' cmd/wego/*.go
 
-bin: ui
+bin:
```

Comment:
After a `make clean`, I think `make bin` will now fail to compile without the UI job since it wants UI files for embedding. Does that still happen on this branch?

Reply:
It will, but that was the only way I could find without having creating something like `docker-bin` or so, since the go stage doesn't have node in it

Reply:
I think its okay as long as this is documented in the README and we let everyone know